tests/lib/k8s/test_builders.py

```diff
@@ -42,6 +42,25 @@ def check_namespace(parameters):
             "namespace": check_namespace,
         }
 
+    @patch("nestor_api.lib.k8s.builders.io", autospec=True)
+    def test_load_templates(self, io_mock):
+        """Should load the templates and correctly substitute `{{variable}}`"""
+
+        def _read_mock(file_name):
```

Comment:
Nit picking: Maybe a lambda would improve the readability here

Reply:
I tried and it looked indeed better 👍